test/e2e/windows/gmsa_full.go

```diff
@@ -34,7 +34,7 @@ limitations under the License.
 //    the webhook's pod.
 //  * in order to run "can read and write file to remote folder" test case, a folder (e.g. "write_test") need to be created
 //    in that AD domain and it should be shared with that GMSA account.
-// All these assumptions are fulfilled by an AKS extension when setting up the AKS
```

Comment:
LOL. i think it is AKS :)

please go through once line by line.

Reply:
actually I did @dims , missed this though

Reply:
I have nullified the change @dims . ptal